src/NimbusInitialSale.sol

```diff
@@ -120,7 +120,7 @@ interface INBU {
     function balanceOf(address account) external view returns (uint256);
     function transfer(address recipient, uint256 amount) external returns (bool);
     function approve(address spender, uint256 amount) external returns (bool);
-    function give(address recipient, uint256 amount) external;
+    function give(address recipient, uint256 amount) external returns (bool);

```

Comment:
This change has not been reflected in the NBU.sol file, no? If so, that would mean the re-deployment of NBU and the re-emission of these tokens which have already been given to phase 1 users

Reply:
Good catch Pedro! @nimbus-devs, can we get any updates on this one? Thanks